commands/runner.go

```diff
@@ -188,3 +198,26 @@ func splitAliasCmd(cmd string) ([]string, error) {
 
 	return words, nil
 }
+
+func getListOfBuiltInCommands() map[string]bool {
+	gitHelpOutput, _ := cmd.New("git help -a").CombinedOutput()
+	builtInCommands := parseGitHelpOutput(gitHelpOutput)
+	for hubCommand, _ := range CmdRunner.All() {
+		builtInCommands[hubCommand] = true
```

Comment:
It's not necessary that we write custom commands into `builtInCommands` hash just for the purpose of the check. Instead, `isBuiltInCommand` could iterate over each command in `CmdRunner.All()` and return `true` early if any command matched. Otherwise, it should proceed getting a list of commands from `git help -a`.